politeiawww/api/pi/v1/v1.go

```diff
@@ -535,8 +535,54 @@ type VoteAuthorizeReply struct {
 	Receipt   string `json:"receipt"`
 }
 
-type VoteStart struct{}
-type VoteStartReply struct{}
+// VoteOption describes a single vote option.
+type VoteOption struct {
+	ID          string `json:"id"`          // Single, unique word (e.g. yes)
+	Description string `json:"description"` // Longer description of the vote
+	Bit         uint64 `json:"bit"`         // Bit used for this option
+}
+
+// VoteDetails includes all data required by server to start vote on a
+// proposal.
+type VoteDetails struct {
+	Token    string `json:"token"`    // Record token
```

Comment:
Should be refered to as a proposal when talking about it in the context of the pi api.  Apply to the other comments as well.